iron-doc-property.html

```diff
@@ -182,8 +182,8 @@
         },
 
         // hidden if no params, and no return value
-        _computeHideParams: function(descriptor,ret) {
-          return (!descriptor.params || descriptor.params.length === 0) && !ret;
```

Comment:
`this.return` was never actually ever defined and this function always had true for !ret. After looking back at this, I believe that they meant to bind `descriptor.return`. will update